packages/captp/test/traplib.js

```diff
@@ -0,0 +1,141 @@
+// @ts-check
+/* global setTimeout */
+
+import { assert, details as X } from '@agoric/assert';
+import { Far } from '@agoric/marshal';
+import { E, makeCapTP } from '../lib/captp';
+
+export function createHostBootstrap(makeTrapHandler) {
```

Comment:
```suggestion
export const createHostBootstrap = makeTrapHandler => {
```
etc (gotta start sometime)